kubernetes/run-unit-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+mkdir "/srv/http/concent_api/concent_api/settings/config_map"
+touch "/srv/http/concent_api/concent_api/settings/config_map/local_settings.py"
```

Comment:
Please add a comment that explains why we need to do this:

```
# It seems that Django tries to import the default settings module even if it's told to use a different one.
# To work around it we need empty local_settings.py so that importing does not fail.
```

Reply:
Done.